docs/README.md

```diff
@@ -27,7 +27,9 @@ As of v2.3.0, kube-state-metrics supports additional opt-in metrics via the CLI
 
 ## Exposed Metrics
 
-Per group of metrics there is one file for each metrics. See each file for specific documentation about the exposed metrics:
+Per group of metrics there is one file for each metrics. 
+Some resources (ClusterRole, ClusterRoleBinding, Role, RoleBinding and ServiceAccount) are not exposed by default.
```

Comment:
These aren't the only metrics not exposed by default, there are also VPA metrics and maybe some others. Maybe we should make two lists, one for metrics exposed by default and another one with the additional ones.

Reply:
Good point, I split it up in two lists.